ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java

```diff
@@ -244,7 +245,7 @@ private static String normalizeUuid(String v) {
     try {
       // Will raise an exception if not a valid UUID.
       UUID.fromString(v);
-      return v;
+      return v.toLowerCase();
```

Comment:
This will also affect `impression_id`s. But I guess UUIDs should be lowercase in general

Reply:
Yes, I have no objection to lowercasing all UUIDs that pass through this.